bower.json

```diff
@@ -20,7 +20,7 @@
     "keen-js": "~3.4.0",
     "trackjs": "~2.1.16",
     "typeahead.js": "~0.11.1",
-    "plyr": "1.6.x",
+    "plyr": "3.2.0",
```

Comment:
Just updating this file doesn't update the version number, you'd also need to run `bower install`.
BUT, we don't want to update the version of `plyr` because the theme in newer will break our layout since the controls show the background of the video behind them.

Reply:
Okay, i will try to find another work around or if you think this issue cannot be resolved, should it be closed ?

Reply:
@chaymafourati the fix should be a pure CSS change.
I would update `css/styles.cs` with a new rule, something like

```
.plyr__time {
  /** Your style rules here **/
}
```

If it doesn't take effect, add `!important` for now then we can work on getting the fix in without using it.